composer.json

```diff
@@ -131,7 +131,8 @@
         "patches": {
             "acquia/acquia-cms-starterkit": {
                 "ACMS-1988 - Update admin theme in starterkit to Gin": "https://github.com/acquia/acquia-cms-starterkit/compare/develop...feature/admin-gin-theme.patch",
-                "ACMS-3456: Update logic to enable theme.": "https://patch-diff.githubusercontent.com/raw/acquia/acquia-cms-starterkit/pull/132.patch"
+                "ACMS-3456: Update logic to enable theme.": "https://patch-diff.githubusercontent.com/raw/acquia/acquia-cms-starterkit/pull/132.patch",
+                "ACMS-4377: Add sitestudio_gin module in ACMS.": "https://patch-diff.githubusercontent.com/raw/acquia/acquia-cms-starterkit/pull/135.patch"
```

Comment:
Not sure whether we should include these patches, CI is passing I guess once vishal review and approve this, we should remove these patches.

Reply:
The same procedure is followed: once the pull request against the Acquia CMS Starter Kit is merged, the corresponding patch is removed.